janusgraph-core/src/main/java/org/janusgraph/diskstorage/indexing/IndexFeatures.java

```diff
@@ -35,11 +35,12 @@
     private final ImmutableSet<Mapping> supportedStringMappings;
     private final String wildcardField;
     private final boolean supportsNanoseconds;
+    private final boolean supportCustomAnalyser;
```

Comment:
two minor comments here
1. please use the same naming convention as supportsNanoseconds
2. [Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-analyzers.html) spelling here and elsewhere: supportsCustomAnalyzer